scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -183,11 +183,19 @@ def test_rank_deficient(self):
     @pytest.mark.parametrize("n", [8, 15, 21])
     @pytest.mark.parametrize("radius", [0.5, 1, 2])
     @pytest.mark.parametrize("center", [(0, 0, 0), (1, 2, 3)])
-    def test_geodesic_input(self, n, radius, center):
-        U = Rotation.random(random_state=0).as_matrix()
+    def test_circular_input(self, n, radius, center):
```

Comment:
It might be easier to review if the old geodesic tests remain the same & new tests are added for the more general circular input.

Maybe not crucial though---I'll just have to think more carefully when reviewing I guess.

Reply:
Which do you prefer? The new tests generalize the old tests, so I would like to remove the old ones.

Reply:
> The new tests generalize the old tests

Correct me if I'm wrong, but won't `circle_radius` always be smaller than `radius` here? While I realize that's the new feature being added, doesn't this mean that we aren't just generalizing, but now also explicitly not including the geodesic originally tested?

Reply:
You are absolutely correct. This line of inquiry also revealed some logic flaws which I have fixed.